### DISCUSSION TITLE
Problème avec les fichiers xslx ils ne sont plus présent lorsque nous reprenons la saisie d'un formulaire

### DISCUSSION THREAD

**Discussion by MJMESS** [MTESSDev/FRW#44] [Annonces]
Bonjour, 

voici comment reproduire la situation : 

1. Je débute un formulaire
2. Je téléverse des fichiers dont certain de types xlsx
3. Je quitte
4. Je reprend le formulaire,
5. Tous les fichiers de type xlsx ne sont plus présent, un message d'erreur s'affiche.

![image](https://user-images.githubusercontent.com/122536311/229601349-4493d5f1-0e7b-48cb-bc44-6e60d9f7a6ed.png)

Merci à l'avance!

**Comment by j0ber** [comment]
S.V.P. Créer un issue car il semble s'agir d'un bug.
![image](https://user-images.githubusercontent.com/26974817/229603410-ce0fa3d8-165f-4d69-99e7-9510a61db139.png)

**Comment by j0ber** [comment]
Le problème a été corrigé.

C'est dans la release [2023.3](https://github.com/MTESSDev/FRW/releases/tag/2023.3-IT) (Déjà disponible dans l'environnement IT que vous utilisez).

Voir #47
